scss/_patterns_links.scss

```diff
@@ -70,6 +70,7 @@ $external-icon-size--headings: #{map-get($icon-sizes, heading-icon--x-small)};
       background-color: currentColor;
       content: '\00A0';
       display: inline-block;
+      line-height: 1em;
```

Comment:
What does that have to do with this PR?

Reply:
Oh, I thought it was a fix for external icon size. Because I think these icons used to be a bit bigger, and there is some weird bug with size of the CSS map.

Seems to be a different issue.

Anyway, it's OK to fix some drive bys when they are related to issue in place, unrelated drive bys make it confusing - during review, what should be tested; they may accidently cause regressions (because were not QAed properly becaused seemed small), later they are harder to track. In 6 months we may wonder why line-height is 1em, if we track history, we see this in completely unrelated PR and we don't know why it happened.

I know it's a small thing. But if it is a small thing, the easier it should be to create a small, dedicated PR to properly review and QA it.

Reply:
you're right, I'll move it to the correct pr

Reply:
Thanks, could you reset this branch to previous commit and force push again?

`git reset --hard 3131fe7` (while in the `list-fix` branch)